vizro-core/src/vizro/models/_components/form/checklist.py

```diff
@@ -27,6 +27,9 @@ class Checklist(VizroBaseModel):
     title: Optional[str] = Field(None, description="Title to be displayed")
     actions: List[Action] = []
 
+    # Component properties for actions and interactions
+    _input_property: str = PrivateAttr("value")
+
```

Comment:
There's a big correlation between  `_input_property` and `_action_validator_factory`.  

Just to remind that:
- `_action_validator_factory  argument` ("value" in this scenario) represents a default property when the model object is the `trigger` of the actions chain. 
- `_input_property  argument` ("value" in this scenario) represents a default property when the model object is the `state` of some action.